src/components/Menu.js

```diff
@@ -165,34 +168,38 @@ function ResourcesDialog(props) {
         <Dialog {...props}>
             <DialogTitle id="about-dialog-title">{"Resources"}</DialogTitle>
             <DialogContent>
-                <DialogContentText id="about-dialog-description">
+                <DialogContentText
+                    id="about-dialog-description"
+                    component="div"
+                >
                     <p>
                         There are two other websites for the state of
                         Massachusetts that compile information on vaccine
                         availability. They are:
-                        <ul>
-                            <li>
-                                <a
-                                    href="https://vaccinatema.com"
-                                    target="_blank"
-                                    rel="noreferrer"
-                                >
-                                    Vaccinate MA
-                                </a>{" "}
-                                (volunteer-run)
-                            </li>
-                            <li>
-                                <a
-                                    href="https://vaxfinder.mass.gov"
-                                    target="_blank"
-                                    rel="noreferrer"
-                                >
-                                    Vax Finder
-                                </a>{" "}
-                                (state-run)
-                            </li>
-                        </ul>
                     </p>
+
+                    <ul>
```

Comment:
Why are these showing up as different?  Are you using different whitespace?  Is there a way to minimize the changes?  Thanks!

Reply:
One of the warnings was `Warning: validateDOMNesting(...): <ul> cannot appear as a descendant of <p>.`, so I moved the unordered list out of the paragraph. There are resulting whitespace changes because I am running Prettier on Save.

If you check "Hide whitespace changes" in the settings menu, it will make the change a little easier to read.

![image](https://user-images.githubusercontent.com/2482343/108613143-34fde600-73bd-11eb-86f4-2eff5b0ad2bb.png)